Uanion/Uanion.Application/Features/Dialoge/Commands/CreateDialoge/CreateDialogeCommandValidator.cs

```diff
@@ -0,0 +1,28 @@
+﻿using FluentValidation;
+
+namespace Uanion.Application.Features.Dialoge.Commands.CreateDialoge;
+
+public class CreateDialogeCommandValidator : AbstractValidator<CreateDialogeCommand>
+{
+    public CreateDialogeCommandValidator()
+    {
+        RuleFor(p => p.Email)
```

Comment:
This is a validator for CreateDialogCommand. There are no properties in CreateDialogCommand yet, therefore there should be no logic in validator for now